lib/resources/r4/observation.yaml

```diff
@@ -117,6 +178,55 @@ fields:
     }
   url: https://hl7.org/fhir/R4/observation-definitions.html#Observation.code
 
+- name: code
+  required: 'Yes'
+  type: CodeableConcept
+  description: Describes what was observed. Sometimes this is called the observation "name".
+  action:
+    - terminology
+    - update
+  binding:
+    description: Codes identifying names of simple observations.
+    terminology:
+    - display: LOINC
+      system: http://loinc.org
+      info_link: https://hl7.org/fhir/R4/loinc.html
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+      info_link: https://hl7.org/fhir/R4/snomedct.html
+    - display: Millennium Clinical Event Code
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/72
+      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-72-clinical-event-codes
+  example: |
+    {
+      "code": {
+        "coding": [
+          {
+            "system": 'http://loinc.org',
+            "code": '8331-1'
+          }
+        ],
+        "text": 'Temperature Oral'
```

Comment:
This still have vital sign examples. We need lab examples here and below.

Reply:
updated: https://github.com/cerner/fhir.cerner.com/commit/24cb43e7f384892b8a8dbb1aaf51ec9703f2b949#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR205-R208